data/pages-content/cfp.md

```diff
@@ -0,0 +1,72 @@
+---
+title: CFP
+subtitle: EuroPython Call for Proposals
+---
+# Call for Proposals
+
+🐍 Pythonistas, mark the dates:
+
+<div style={{textAlign: "center"}}>
+**EuroPython 2023 Call for Proposals (CFP) will be open** between
+
+ **Monday, 6 March 2023** and
+
+ **Sunday, 19 March 2023** [AoE](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
+</div>
+
+
+
+We invite proposals from the whole community on a wide range of topics. We welcome speakers at all python levels with any experience with public speaking.
+
+## The Conference Format ##
+
+The conference will feature around 16 Tutorials/Workshops and 120 talk sessions:
+
+* **Tutorials/Workshops (17-18 July)**: hands-on **180-mins sessions**
+
+* **Talks (19-21 July)** : **30, 45, or 60-minute** presentations of Python general topics & PyData related topics
+
+All presentation content can be at **beginner, intermediate or advanced** level.
+
+[comment]: # (Other than talks and tutorials, you are also welcome to submit **posters**: printed in large formats, it is a graphical way to showcase a project or a technology. Posters are exhibited at the conference, read at any time by participants, and can be discussed face to face with their authors during the poster session.)
+
+
+## Topics ##
+
+We welcome presentations focusing on a variety of topics all around Python, e.g. programming, engineering, cloud, DevOps, web, Django, hardware, community, education, or whatever you deem suitable - **it is your conference!**
+
+To see the type of topics presented at previous EuroPython events, check out our [past EuroPython conferences](https://www.europython-society.org/europython/).
+Browse through our past [talk videos](https://www.youtube.com/c/EuroPythonConference) for more inspiration.
+
+For PyData related ideas, take a look at PyData events: [pydata.org](https://pydata.org/).
+
+### Open Source ###
+As a reminder, EuroPython presentations are intended to share knowledge and experience. To this end, we encourage the code and/or data that your talk relies on to be open-source. Ideally, the audience would have access to the necessary tools to reproduce the results of the talk. We welcome talks focused on the practical application of tools and concepts you use; but we discourage sales-oriented proposals whose main aim is to sell a product.
+
+## Selection Process ##
+
+We believe that a good conference consists of popular talks as well as new or niche topics - all by and for our diverse community. Thus, after Call for Proposals (CFP) finishes, submissions go through two rounds of review and refinement as we put together our programme. This is a "blind" process, so speakers' names will not be displayed to the reviewers.
+
+* **community voting**: open to all 2015-2023 EuroPython conference ticket holders + 2023 proposal submitters. Submissions are in random order for each voter. Please check out our talk voting page for details.
```

Comment:
I'd either remove the last line "lease check out our talk voting page for details." or replace it with something along the lines of we'll share the URL to talk voting soon.

Reply:
To ensure our programme reflects our diverse community, submissions go through three rounds of review and refinement before the final programme is announced:

1. **Community voting** - is open to all 2015-2023 EuroPython conference ticket holders, and anyone who proposes a talk for the 2023 conference. This helps us judge both the uniqueness, niche-ness, popularity and interestingness of each proposal.
2. **Panel feedback** - a diverse group of experienced reviewers refine and provide constructive feedback on submissions within their realm of expertise. This helps us to judge the balance, quality and diversity of our programme.
3. **Programme team** - appointed by the EuroPython society, these volunteers curate the final programme based upon the results of community voting and panel feedback. They also ensure we meet editorial criteria such as diversity, depth and breadth of topics and speakers.